lib/src/audio_file_waveforms.dart

```diff
@@ -8,57 +8,64 @@ import '../audio_waveforms.dart';
 import 'base/constants.dart';
 
 class AudioFileWaveforms extends StatefulWidget {
-  ///Height and width of waveform.
+  /// A size to define height and width of waveform.
   final Size size;
 
-  ///Use this control the waveform.
+  /// A PlayerController having different controls for audio player.
   final PlayerController playerController;
 
-  ///Use this to style the waveform.
+  /// A PlayerWaveStyle object control how waveforms should look.
   final PlayerWaveStyle playerWaveStyle;
 
-  ///Use this to give padding around waveform.
+  /// Provides padding around waveform.
   final EdgeInsets? padding;
 
-  ///Use this to give margin around waveform.
+  /// Provides margin around waveform.
   final EdgeInsets? margin;
 
-  ///Use this to decorate background of waveforms
+  /// Provides box decoration container having waveforms.
   final BoxDecoration? decoration;
 
-  ///background color of waveform. if decoration is used then use color in it.
+  /// Color which is applied in to background of the waveform.
+  /// If decoration is used then use color in it.
   final Color? backgroundColor;
 
-  ///Enable/Disable seeking using gestures. Defaults to true.
+  /// Enable/Disable seeking using gestures. Defaults to true.
   final bool enableSeekGesture;
 
-  ///Duration for animation. Defaults to 500 milliseconds.
+  /// Duration for animation. Defaults to 500 milliseconds.
   final Duration animationDuration;
 
-  ///Curve for animation. Defaults to Curves.bounceOut
+  /// Curve for animation. Defaults to Curves.bounceOut
   final Curve animationCurve;
 
-  ///Density of the display. Providing accurate density is not neccesary, if desired looking
-  ///waveforms are needed.
+  /// Density of the display. Providing accurate density is not necessary,
+  /// if desired looking waveforms are needed.
   ///
```

Comment:
remove this extra comment lines

Reply:
It provides new line for next comment so it's required